dev_guide/builds.adoc

```diff
@@ -10,21 +10,20 @@
 toc::[]
 
 == Overview
-A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build]
-is a process of creating runnable images to be used on OpenShift. There are
-three build strategies:
+A link:../architecture/core_concepts/builds_and_image_streams.html#builds[build] is a process of creating
+runnable images to be used on OpenShift. There are three build strategies:
 
-- link:../architecture/core_concepts/builds_and_image_streams.html#source-build[Source-To-Image]
-- link:../architecture/core_concepts/builds_and_image_streams.html#docker-build[Docker]
-- link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[Custom]
+-link:../architecture/core_concepts/builds_and_image_streams.html#source-build[Source-To-Image]
+-link:../architecture/core_concepts/builds_and_image_streams.html#docker-build[Docker]
+-link:../architecture/core_concepts/builds_and_image_streams.html#custom-build[Custom]
```

Comment:
Looks like these three lost the space after the `-`, so they're not rendering as a list.